src/main/java/net/ddellspe/music/bot/commands/PrefixMessageResponseCommand.java

```diff
@@ -10,4 +11,10 @@
   Snowflake getFilterChannel(Snowflake guildId);
 
   Mono<Void> handle(MessageCreateEvent event);
+
+  default String getMessageAfterPrefix(MessageCreateEvent event) {
+    MusicAudioManager manager = MusicAudioManager.of(event.getGuildId().get());
+    String prefix = manager.getPrefix() + getName() + ' ';
+    return event.getMessage().getContent().split(prefix)[1];
```

Comment:
this might throw an out of range exception, need to look into that with testing.